rcl/src/rcl/validate_topic_name.c

```diff
@@ -213,7 +215,7 @@ rcl_topic_name_validation_result_string(int validation_result)
     case RCL_TOPIC_NAME_INVALID_SUBSTITUTION_STARTS_WITH_NUMBER:
       return "substitution name must not start with a number";
     default:
-      return NULL;
+      return "undefined topic name type";
```

Comment:
This one is a bit obscure IMO. Would something like "unknown 'rcl_validate_topic_name' result code" be clearer?